backend/log/log.go

```diff
@@ -5,12 +5,15 @@ import (
 	hclog "github.com/hashicorp/go-hclog"
 )
 
+type Level hclog.Level
+
 // Logger is the main Logger interface.
 type Logger interface {
 	Debug(msg string, args ...interface{})
 	Info(msg string, args ...interface{})
 	Warn(msg string, args ...interface{})
 	Error(msg string, args ...interface{})
+	Level() Level
```

Comment:
Does this require external plugins to take a dependency on hclog?

Reply:
I assumed not since we define that type here?

Reply:
@marefr changed this a bit to add constants for the levels